include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp

```diff
@@ -66,6 +62,36 @@ class turn_info_exception : public geometry::exception
 namespace detail { namespace overlay
 {
 
+
+struct policy_verify_nothing
+{
+    static bool const use_side_verification = false;
+    static bool const use_start_turn = false;
```

Comment:
should `use_start_turn`, `use_handle_as_touch`, `use_handle_as_equal` and `use_handle_imperfect_touch` be merged to one?

Reply:
Good question. I was not sure yet. Yes, we could merge them all - but basically linestrings could profit from these too (to be investigated), and these settings make it easy to gradually change them

Reply:
I'll keep it for now, it's internal, we can revisit it later.